packages/design-system-docs/src/pages/components/UsaBanner/_UsaBanner.docs.scss

```
@@ -0,0 +1,45 @@
+/*
+USA Banner
```

Let's update to sentence case 
```suggestion
USA banner
```
We don't consistently use sentence case but should be as it'a bit easier to read. 
I'm going to file a ticket to update page titles to use sentence case